internal/cmdupdate/cmdupdate.go

```diff
@@ -75,7 +77,35 @@ func (r *Runner) preRunE(c *cobra.Command, args []string) error {
 	if len(parts) > 2 {
 		return errors.Errorf("at most 1 version permitted")
 	}
-	r.Update.Path = parts[0]
+
+	cwd, err := os.Getwd()
```

Comment:
How about encapsulating this logic into a library as it might be useful to other commands like `get`?

Reply:
I think we can do that if we see the need.

Reply:
Done